docs/elementaloperatorchart-reference.md

```diff
@@ -0,0 +1,78 @@
+# Elemental Operator Helm Chart
+
+The [{{elemental.operator.name}}]({{elemental.operator.url}}) is responsible for managing the Elemental versions and maintaining a machine inventory to assist with edge or baremetal installations.
+
+The associated chart bootstraps an elemental-operator deployment on the [Rancher Manager v2.6](https://rancher.com/docs/rancher/v2.6/) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Rancher Manager version v2.6
+- Helm client version v3.8.0+
+
+## Get Helm chart info
+
+```console
+helm pull oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+helm show all oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+```
+
+## Install Chart
+
+```console
+helm install --create-namespace -n cattle-elemental-system elemental-operator \
+             oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+```
+
+The command deploys elemental-operator on the Kubernetes cluster in the default configuration.
+
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
+
+## Uninstall Chart
+
+```console
+helm uninstall -n cattle-elemental-system elemental-operator
+```
+
+This removes all the Kubernetes components associated with the chart and deletes the release.
+
+_See [helm uninstall](https://helm.sh/docs/helm/helm_uninstall/) for command documentation._
+
+## Upgrading Chart
+
+```console
+helm upgrade -n cattle-elemental-system \
+             --install elemental-operator \
+             oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+```
+
+_See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
+
+## Configuration
+
+See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values](#values), or run these configuration commands:
+
+```console
+helm show values oci://registry.opensuse.org/isv/rancher/elemental/charts/elemental/elemental-operator
+```
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| image.empty | string | `rancher/pause:3.1` |  |
```

Comment:
This is missing the `replicas`, probably the source docs on the operator are not up to date :/

In any case, I dont think this is necessary, when @nunix changes the docs he mentioned that they will be together so we probably dont need to include the operator docs in elemental repo

Reply:
I have also been wondering about it too. The problem with the current status is the Readme is not published anywhere, so from docs pages you can't reach this information. That's why I thought about adding it here, I was also thinking about deleting from the elemental-operator repo so no duplicated code and just refer to docs in elemental-operator Readme.